src/client/auth/mod.rs

```diff
@@ -437,6 +437,14 @@ impl Credential {
     }
 }
 
+impl Debug for Credential {
```

Comment:
This and the added test are the meat of the changes. There were also some unrelated clippy updates that were made on the master branch but not this one.